src/frontends/pytorch/src/op/topk.cpp

```diff
@@ -0,0 +1,44 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <climits>
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/opsets/opset10.hpp"
```

Comment:
Could you please use individual operation includes instead of big opset include to speed up build.